playground/components/framework-switcher.tsx

```diff
@@ -0,0 +1,41 @@
+'use client';
+
+import { Frameworks } from '@/types';
+import { useSandpack } from '@codesandbox/sandpack-react';
+import { Dispatch, SetStateAction } from 'react';
+
+export function FrameworkSwitcher({
+  setFramework,
+}: {
+  setFramework: Dispatch<SetStateAction<Frameworks>>;
+}) {
+  const { dispatch } = useSandpack();
+
+  const handleClick = (framework: Frameworks) => {
+    setFramework(framework);
+    setTimeout(() => {
+      dispatch({ type: 'shell/restart' });
+    }, 1000);
+  };
+
+  return (
+    <>
+      <div className="flex flex-col items-center justify-center h-12">
+        <div className="flex items-center justify-center">
+          <button
```

Comment:
I would change two things here:

- This should be a select instead of buttons. This will not scale nicely to more frameworks 
- We should dynamically render those based on a configuration object or something where editing will be easier and dynamic in the future.

Reply:
Nice!